.github/workflows/ci.yml

```diff
@@ -36,7 +36,53 @@ jobs:
         run: poetry run pytest .
         env:
           CO_API_KEY: ${{ secrets.COHERE_API_KEY }}
-  
+
+  test-langchain-cohere:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v3
+      - name: Set up python
+        uses: actions/setup-python@v4
+        with:
+          python-version: 3.10.14
+      - name: Bootstrap poetry
+        run: |
+          curl -sSL https://install.python-poetry.org | python - -y --version 1.5.1
```

Comment:
Can we use a pinned version of https://github.com/marketplace/actions/install-poetry-action instead?